packages/common/src/waku/waku-observers.ts

```diff
@@ -25,8 +26,8 @@ export class WakuObservers {
   private static currentChain: Optional<Chain>;
   private static currentContentTopics: string[] = [];
   private static currentSubscriptions:
-    | { subscription: IFilterSubscription; params: SubscriptionParams[] }[]
-    | undefined;
+    | { subscription: Unsubscribe; params: SubscriptionParams[] }[]
```

Comment:
if just an array of unsubscriptions, should it be `unsubscription: Unsubscribe`? And if just storing the unsubscription, I'm guessing the poller does not ping the subscriptions themselves since they aren't stored

Reply:
this is just how its named, due to the object creating it. its creating a 'subscription' that in turn when you call it 'unsubscribes' to the subscription. this is no longer necessary and handled by the waku-core i believe.

Reply:
Edit: I believe in the latest SDK, createSubscription no longer exists, and it seems to want only use of 'subscribe' which returns a subscription object itself now, which can then be managed